src/main/java/fr/xephi/authme/listener/EntityListener.java

```diff
@@ -81,6 +80,7 @@ public void entityRegainHealthEvent(EntityRegainHealthEvent event) {
         }
     }
 
+    //TODO sgdc3 20190808: Does it still make sense?
```

Comment:
i don't know why we listen at the same event with 2 different priorities, the only thing i can think of is that we want to prevent another plugin from un-cancelling the event :/